packages/flutter_tools/templates/create/ios.tmpl/Flutter/Debug.xcconfig

```diff
@@ -0,0 +1,2 @@
+#include "Generated.xcconfig"
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```

Comment:
These files are normally in the Pods directory, which I need to check in anyway since we don't want to require people to install CocoaPods to use flutter create. I've added it to this pull request.